wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -82,6 +82,9 @@ class SwiftGenerator private constructor(
   private val encoder = DeclaredTypeName.typeName("Swift.Encoder")
   private val writableKeyPath = DeclaredTypeName.typeName("Swift.WritableKeyPath")
 
+  // Always fully-qualified in case a message type is named `Data`
+  private val data = DeclaredTypeName.typeName("Foundation.Data", true)
```

Comment:
I initially tried searching through the `schema` to see if one of the defined messages was named `Data`, but I couldn't quite figure that out